client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -163,14 +167,12 @@ const PluginsBrowser = ( { trackPageViews = true, category, search, hideHeader }
 				}
 			/>
 
-			{ ! hideHeader && (
```

Comment:
I removed this prop after checking that it wasn't used anywhere on the codebase.